src/video/groupnetcamview.cpp

```diff
@@ -199,14 +198,16 @@ void GroupNetCamView::onUpdateActivePeer()
     setActive();
 }
 
-void GroupNetCamView::setActive(int peer)
+void GroupNetCamView::setActive(ToxPk peer)
 {
-    if (peer == -1) {
+    if (peer == ToxPk{}) {
```

Comment:
Probably better will be add a method `isEmpty` in ToxPk

Reply:
switched to `isEmpty`